lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -490,6 +563,14 @@ def _create_node_os_disk(
     ) -> None:
         node_context = get_node_context(node)
 
+        if self.qemu_platform_runbook.is_host_remote():
```

Comment:
You should set the `host_node` to the same value as `local_node` when running on the local machine. That way, this function can use identical code for the local and remote cases.

Reply:
Once the node is created, it doesn't need to differentiate remote or local node. Use the `Node` type to get consistent behavior. The only difference is the remote node need to `set_connection_info`.